vgstation13.dme

```diff
@@ -2859,7 +2859,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_multiz.dm"
```

Comment:
This will change the default to the test map.

Reply:
i always do this, fixed